shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/rql/resource/DataSourceQueryResultSet.java

```diff
@@ -72,18 +90,66 @@ public boolean next() {
     @Override
     public Collection<Object> getRowData() {
         String dataSourceName = dataSourceNames.next();
+        DataSourceProperties dataSourceProperties = dataSourcePropsMap.get(dataSourceName);
+        DataSourcePoolMetaData poolMetaData = DataSourcePoolMetaDataFactory.newInstance(dataSourceProperties.getDataSourceClassName());
+        Map<String, Object> allProperties = getAllProperties(resource.getDataSources().get(dataSourceName));
+        Map<String, Object> standardProps = getStandardProperties(dataSourceProperties, poolMetaData, allProperties);
+        return getRowData(dataSourceName, poolMetaData, allProperties, standardProps);
+    }
+    
+    private Collection<Object> getRowData(final String dataSourceName, final DataSourcePoolMetaData poolMetaData, final Map<String, Object> allProperties, final Map<String, Object> standardProps) {
         DataSourceMetaData metaData = resource.getDataSourcesMetaData().getDataSourceMetaData(dataSourceName);
-        return Arrays.asList(dataSourceName, resource.getDatabaseType().getName(), metaData.getHostname(), metaData.getPort(), metaData.getCatalog(), 
-                new Gson().toJson(getFilteredUndisplayedProperties(dataSourcePropsMap.get(dataSourceName))));
+        Collection<Object> result = new LinkedList<>();
+        result.add(dataSourceName);
+        result.add(resource.getDatabaseType().getName());
+        result.add(metaData.getHostname());
+        result.add(metaData.getPort());
+        result.add(metaData.getCatalog());
+        result.add(getStandardProperty(standardProps, CONNECTION_TIMEOUT_MILLISECONDS));
+        result.add(getStandardProperty(standardProps, IDLE_TIMEOUT_MILLISECONDS));
+        result.add(getStandardProperty(standardProps, MAX_LIFETIME_MILLISECONDS));
+        result.add(getStandardProperty(standardProps, MAX_POOL_SIZE));
+        result.add(getStandardProperty(standardProps, MIN_POOL_SIZE));
+        result.add(getStandardProperty(standardProps, READ_ONLY));
+        result.add(new Gson().toJson(getFilteredUndisplayedProperties(poolMetaData, allProperties, standardProps)));
+        return result;
+    }
+    
+    private Map<String, Object> getAllProperties(final DataSource dataSource) {
+        DataSourceReflection dataSourceReflection = new DataSourceReflection(dataSource);
+        Map<String, Object> result = dataSourceReflection.convertToProperties();
+        return result;
```

Comment:
Just use `DataSourcePropertiesCreator.create()` to create DataSourceProperties, which include all processed properties.
Please do not call `DataSourceReflection` directly, which is for DataSourceProperties's internal call only.